pkg/server/models.go

```diff
@@ -0,0 +1,17 @@
+package server
+
+type ClusterNamespaceMap struct {
+	ClusterNamespace map[string]string
+	//map[string]in64 --> ex: {"ns_local-cluster_open-cluster-management-observability": 12343212}
+}
+
+type RecommendationIDMap struct {
+	RecommendationID map[string]string
+	//map[int64]string --> ex. {12343212: "ns_local-cluster_open-cluster-management-observability_deployment_container"
+}
+
+type RecommendationStatusMap struct {
```

Comment:
Can we change the names of the structs here . Also do we need 3 different Structs here ? Or can can hold everything in one struct ?

Reply:
Done.